src/components/tab-nav/tab-nav.e2e.ts

```diff
@@ -9,6 +9,25 @@ describe("calcite-tab-nav", () => {
 
   it("is accessible", async () => await accessible(tabNavHtml));
 
+  it("emits on user interaction", async () => {
+    const page = await newE2EPage();
+    await page.setContent(html`<calcite-tab-nav>
+      <calcite-tab-title active>Tab 1 Title</calcite-tab-title>
+    </calcite-tab-nav>`);
+    const activeEventSpy = await page.spyOnEvent("calciteTabChange");
+    const firstTabTitle = await page.find("calcite-tab-title");
+
+    firstTabTitle.setProperty("active", true);
```

Comment:
Since the `active` prop is displayed in the initial html, is this line needed? ☝️